set-web/src/main/java/eu/europeana/set/web/model/vocabulary/UserRoles.java

```diff
@@ -0,0 +1,74 @@
+package eu.europeana.set.web.model.vocabulary;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import eu.europeana.api.commons.definitions.vocabulary.Role;
+
+public enum UserRoles implements Role {
```

Comment:
Is this class not a dupplication of the upper Roles enum? I assume that the upper one needs to be deleted

Reply:
This class is deleted. Just this branch is totally corrupted. I moved everything to a new branch: https://github.com/europeana/set-api/tree/%2326_fix_keycloak_issues